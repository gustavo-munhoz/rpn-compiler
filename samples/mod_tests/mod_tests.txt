(10 3 %)
(10 5 %)
(7 3 %)
(3.5 2 %)
(80 15 %)
(-10 3 %)
(10 -3 %)
(-10 -3 %)
(-7 3 %)
(-2.56 4 %)
(0 5 %)
(0 -5 %)
(2 5 %)
(-2 5 %)
(2 -5 %)
(-2 -5 %)
(65504 100 %)
(10 65504 %)
(1.5 0.5 %)