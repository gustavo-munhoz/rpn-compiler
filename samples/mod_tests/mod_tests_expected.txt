0x3C00
0x0000
0x3C00
0x3E00
0x4500
0xBC00
0x3C00
0xBC00
0xBC00
0xC11E
0x0000
0x0000
0x4000
0xC000
0x4000
0xC000
0x4400
0x4900
0x0000